### PR TITLE
Added the brainfck programming language

### PR DESCRIPTION
**Pull request by fehlercode03**
Added the basic operations of the brainf*ck programming language

### CODE REVIEW

**Comment by sillyangel** [comment]
how would it make word with it with the brainf language?

**Comment by sillyangel** [comment]
but nice 👍🏻

**Review by sillyangel — CHANGES_REQUESTED** [review]
you need to remove the extra space

File: frontend/static/languages/_groups.json [hunk 1]
```diff
@@ -310,6 +310,8 @@
       "code_csharp",
       "code_c++",
       "code_dart",
+      "code_brainfck",
+
```
[the remark this answers is not in the document]
Comment (fehlercode03) [reply frontend/static/languages/_groups.json:314]: It was there before so I left it there

**Comment by Miodec** [comment]
Not sure if this will work well since it will always separate these characters with a space

**Comment by fehlercode03** [comment]
> Not sure if this will work well since it will always separate these characters with a space

What I thought about, maybe small code snippets or something like that e.g.: <<<++.

**Comment by Miodec** [comment]
> What I thought about, maybe small code snippets or something like that e.g.: <<<++.

That would work - couple hundred snippets of varying length.

**Review by Miodec — APPROVED** [review]

**Pull request merged**